.rubocop.yml

```diff
@@ -19,8 +19,8 @@ Layout/LeadingCommentSpace:
   AllowRBSInlineAnnotation: true
 
 Layout/LineLength:
+  Enabled: true
```

Comment:
We removed it from the styleguide by design. I don't think any of our projects should enable. The reason to remove is that it just create a lot of `rubocop:disable` in the code with no real gain.

Reply:
Most editors wrap long lines those days. Monitors are also larger.

Reply:
Removed it and cleaned the remaining Rubocop disable comments for it 👍